animations.js

```diff
@@ -1,33 +1,13 @@
-//var el = document.getElementById('calendar')
+let elMenuItems = document.getElementsByClassName('item-link')
 
-//el.onclick = function () {
-//console.log('Click just happened')
-//}
+Array.from(document.getElementsByClassName('item-link')).forEach(function (
+  element,
+  index,
+  array
+) {
+  element.addEventListener('click', () => {
+    element.classList.toggle('active')
```

Comment:
Well done, this is perfect! Now it's just missing the logic to remove the `active` from the remaining items.

Hint:
Option A: inside the event listener, before the toggle, do another loop to remove the class from all the items. 

Option B (more efficient, but more complex): When clicking in an item using the `index` to store the latest clicked item at `activeIndex`. Use that global variable to decide which item.active should be removed in new clicks.

Reply:
I see you opt-in for the Option A. Here's the solution for Option B:

```js
let elMenuItems = document.getElementsByClassName("item-link");

// -1 by default means there's no current element
let activeIndex = -1;

Array.from(elMenuItems).forEach(function (element, index) {
  element.addEventListener("click", () => {
    // If there's an active menu, and it's different from the current
    if (activeIndex > -1 && activeIndex !== index) {
      // ...remove .active from it
      elMenuItems[activeIndex].classList.remove("active");
    }

    // Toggle active to the clicked element
    element.classList.toggle("active");
    activeIndex = index;
  });
});

```